ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -825,12 +825,13 @@ export const DatasetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), wi
     }
 
     selectPrePackagedCohort(): void {
-      // Un-select any workspace Cohort if Pre Packaged cohort is selected
-      if (!this.state.includesAllParticipants) {
-        this.setState({includesAllParticipants: !this.state.includesAllParticipants, selectedCohortIds: []});
-      } else {
-        this.setState({includesAllParticipants: !this.state.includesAllParticipants});
-      }
+      const {includesAllParticipants, selectedCohortIds} = this.state;
+      this.setState({
+        dataSetTouched: true,
```

Comment:
Issue was caused by not setting `dataSetTouched: true` in state when the checkbox changed.